src/types/index.ts

```diff
@@ -78,12 +79,21 @@ export type Drinks = WithText
 
 export type Additive = Topping | Without | Drinks
 
+// export type Without = {
+//   text: string
+//   textru: string
+// }
+
 export type ToppingsManagerState = {
   topping: Map<string, {
     text: string
+    textru: string
     price: number
   }>
-  without: Set<string>
+  without: Map<string, {
+    text: string
+    textru: string
+  }>
```

Comment:
are you sure that those are needed, I mean about Map and Set, cause initially the logics at toppingManager was with Set/Map which was removed when I added context?

Reply:
So i can delete this?